src/main/java/com/gregtechceu/gtceu/client/model/TextureOverrideModel.java

```diff
@@ -61,7 +61,7 @@ public static List<BakedQuad> retextureQuads(List<BakedQuad> quads,
             if (replacement != null) {
                 newQuads.add(GTQuadTransformers.setSprite(quad, replacement));
             } else {
-                newQuads.add(OVERLAY_OFFSET.process(quad));
+                newQuads.add(quad);
```

Comment:
We don't add the overlay offset here because we say that's up to the caller of retextureQuads now?